### ISSUE TITLE
Plugin fails to appear in the selected area

### ISSUE THREAD

**Issue by namake12138** [HeikkiVesanto/QGIS_Centroid_Within#6]
HELLO HEIKKIVESANTO! o(*￣▽￣*)ブ  I'm sorry to bother you.
We tried countless times (including dissovled, fix layer ....) while making the grid_range_select job (and tried another Qgis file). But it would always appear that the selection was successful, but not selected. How are we going to solve this problem?
My qgis version: 3.26.3
My windows version: 10
Looking forward to your answer!

![微信图片_20231223105152](https://github.com/HeikkiVesanto/SelectWithin/assets/154668336/bcdf4466-0b59-4397-ac9e-2ef0deec0b27)

![微信图片_20231223105159](https://github.com/HeikkiVesanto/SelectWithin/assets/154668336/67078357-31f7-424d-b191-c679f796d4a1)

**Comment by HeikkiVesanto** [comment]
Hey,

Do the layers have the same projection system?

**Comment by namake12138** [comment]
Yeah, they're not in the same projection system. Thank you very much for your answer, I solved the problem.
